persistence-modules/spring-persistence-simple-2/pom.xml

```diff
@@ -53,6 +60,8 @@
         <org.springframework.version>5.2.4.RELEASE</org.springframework.version>
         <!-- persistence -->
         <h2.version>1.4.200</h2.version>
+        <!-- simple-jndi  -->
```

Comment:
There is an extra space in the comment text here.

Reply:
Done!